starfish/codebook/codebook.py

```diff
@@ -307,8 +323,17 @@ def from_json(
         """
         backend, name, _ = resolve_path_or_url(json_codebook)
         with backend.read_contextmanager(name) as fh:
-            code_array = json.load(fh)
-        return cls.from_code_array(code_array, n_round, n_ch)
+            codebook_doc = json.load(fh)
+
+        if isinstance(codebook_doc, list):
+            raise ValueError(
+                f"codebook is a list and not an array.  It is highly likely that you are using a "
```

Comment:
should this be "codebook is a list and not a dictionary"?

Reply:
Fixing.